base.rb

```diff
@@ -1,12 +1,12 @@
-def do_something
+def do_something(p)
   
 end
 
-def do_something2(a, b)
+def do_something2(a, b, c, d = 3)
```

Comment:
__[method definition validator]__

- Not enough arguments. Did you forget the following arguments? [c] at https://github.com/pocke/sandbox2/blob/rubocop-definition-validator-pr/use.rb#L2